invenio_communities/assets/semantic-ui/less/invenio_communities/theme.less

```diff
@@ -46,6 +46,9 @@
     margin-left: 5px;
   }
 
+  .right.floated.right.aligned.community-header-actions.column {
```

Comment:
add an `id` instead of a such long class? 😎 

`<div id="community-upload-button" class="four wide right floated right aligned community-header-actions column">`

```
#community-upload-button {
  padding: 0px;
}
```

Reply:
Nice idea :)